lib/jekyll/filters.rb

```diff
@@ -357,6 +368,34 @@ def sort_input(input, property, order)
         .map!(&:last)
     end
 
+    private
+    # month_type: "b", "B"
+    # type: nil, "ordinal"
+    # style nil, "US"
+    def generic_date_to_string(date, month_type, type = nil, style = nil)
```

Comment:
Both `type` and `style` should default to something other than `nil` here, IMHO.

Reply:
`nil` is equivalent to default. We can give it something more complicate, but the code will work anyway with something different. :thinking: